src/factors/PartialPose3.jl

```diff
@@ -19,11 +19,21 @@ end
 
 PriorPose3ZRP(z::T1,rp::T2) where {T1 <: SamplableBelief, T2 <: SamplableBelief} = PriorPose3ZRP{T1,T2}(z, rp)
 
-getManifold(::PriorPose3ZRP) = ProductGroup(ProductManifold(TranslationGroup(1), SpecialOrthogonal(3)))
+# getManifold(::PriorPose3ZRP) = ProductGroup(ProductManifold(TranslationGroup(1), SpecialOrthogonal(3)))
+getManifold(::PriorPose3ZRP) = SpecialEuclidean(3)
 
-#FIXME update
-function getSample(cfo::CalcFactor{<:PriorPose3ZRP})
-  return ([rand(cfo.factor.z); rand(cfo.factor.rp)], )
+#FIXME update to also only one measurement
+function getSample(cf::CalcFactor{<:PriorPose3ZRP})
+  
+  M = getManifold(cf.factor)
+
+  ϵ = identity_element(M)
+
+  X = hat(M, ϵ, [0; 0; rand(cf.factor.z); rand(cf.factor.rp); 0])
```

Comment:
No, think this is wrong.  You first need to convert a roll and pitch from Euler angles to so(3) coordinates -- this step:
https://github.com/dehann/TransformUtils.jl/blob/7b0a1411152e77e690ddb6477d81a39672ded748/src/Legacy.jl#L516

Reply:
Oh, I assumed we wanted to work in so(3), but it makes more sense that it is Euler angles

Reply:
We definitely want the coordinates to be in Lie Exponential around point p.  For these factors specifically we need a way to first convert between Euler angles.

The previous stable code used Euler angles as coordinates, but we want to stop doing that.

Reply:
I updated it to Euler